### PR TITLE
OCLOMRS-55: Fix the loader right before the dictionaries are retrieved

### PR DESCRIPTION
**Pull request by EleisonC**
# JIRA TICKET NAME:
[OCLOMRS-55](https://issues.openmrs.org/browse/OCLOMRS-55)

# Summary:
A fix to show the loader right during the fetching process of dictionaries

### CODE REVIEW

File: src/redux/actions/dictionaries/dictionaryActionCreators.js [hunk 1]
```diff
@@ -28,21 +28,17 @@ export const fetchingOrganizations = () => dispatch =>
...
-          if (error.payload){
-            dispatch(isSuccess(error.payload));
-            dispatch(isFetching(false));
-          } else {
-            dispatch(isErrored(error.payload.data));
-            dispatch(isFetching(false));
-          }
-        }
-        );
+export const fetchDictionaries = () => dispatch =>{
```
Comment (mwaz) [inline src/redux/actions/dictionaries/dictionaryActionCreators.js]: I believe there should be a space between the brace and the arrow function on line 31

File: src/redux/actions/dictionaries/dictionaryActionCreators.js [hunk 2]
```diff
@@ -28,21 +28,17 @@ export const fetchingOrganizations = () => dispatch =>
...
+  dispatch(isFetching(true));
+  return api.dictionaries
+    .fetchingDictionaries()
+    .then(payload => {
+      dispatch(isSuccess(payload));
+      dispatch(isFetching(false));
+    })
+    .catch(error => {
+      let verror = "Can not make this request";
```
Comment (mwaz) [inline src/redux/actions/dictionaries/dictionaryActionCreators.js]: where is this used?

**Comment by mwaz** [comment]
LGTM

File: src/redux/actions/dictionaries/dictionaryActionCreators.js (hunk 1 above)
Comment (EleisonC) [reply src/redux/actions/dictionaries/dictionaryActionCreators.js]: That has been rectified.

File: src/redux/actions/dictionaries/dictionaryActionCreators.js (hunk 2 above)
Comment (EleisonC) [reply src/redux/actions/dictionaries/dictionaryActionCreators.js]: Nowhere and it has been removed.

**Comment by emasys** [comment]
LGTM

**Comment by rubailly** [comment]
LGTM

**Pull request merged**